### DISCUSSION TITLE
CMS pages title translation after update not work

### DISCUSSION THREAD

**Discussion by oim37** [wintercms/winter#1270] [Q&A] [answered]
Hello,

I update winter to 1.2.7 with composer update and see that title, meta title, meta description allways whowing only on default language. when switching languages, they are always displayed in the default language and do not change according to the one specified in the admin panel. Thus, the search engine will calculate еhese pages are duplicates. I think this is a bug.

**Comment by mjauvin** [comment]
please give more details.

**Comment by oim37** [comment]
I have cms pages with title translations.

Default lang title
![Default-russian-lang](https://github.com/user-attachments/assets/8fcd365b-d2d7-4502-981b-9ba10eaf4e0a)

En lang title
![Second-en-lang](https://github.com/user-attachments/assets/2938a606-87c6-4433-a23a-8b4bcacd4a0b)

When i go to domain.tld/404 i see default lang title
When i go to domain.tld/en/404 i see default lang title too, but it must be on english.

And on other 9 languages same. This is only in cms pages. Blog and static pages work good. It after update to last wintercms.

**Comment by mjauvin** [comment]
I confirm this is a regression.

**Comment by mjauvin** [comment]
ref. #1271

**Comment by mjauvin** [comment, marked as answer]
@oim37 can you try latest dev-main branch of Winter.Translate plugin, that should resolve the issue.

You can switch by running the following command in your app's root folder:

`composer require winter/wn-translate-plugin dev-main`

**Comment by oim37** [comment]
Thank you. It fixed. Now working good.

**Comment by oim37** [comment]
And a small wish: roundcube uses this method.
And i think it good solutuons.
I change settings in plugins/winter/translate/config/config.php
to 'prefixDefaultLocale' => env('TRANSLATE_PREFIX_LOCALE', false),
and after update it by default true.

Is it possible to add a custom config, since when updating the main one is replaced and you always need to remember about the settings.

may be add config.custom.php which overrides the default settings. These are just thoughts and wishes.

**Comment by mjauvin** [comment]
@oim37 Plugin's config can be overriden in your App's root folder.

ref. https://wintercms.com/docs/v1.2/docs/plugin/settings#file-based-configuration

quote:

> A plugin configuration can be overridden by the application by creating a configuration file config/author/plugin/config.php, for example config/acme/todo/config.php, or config/acme/todo/dev/config.php for an environment specific override (in this case dev).

**Reply by oim37** [reply]
Got it, thanks.
